api_v2/src/leaves/leaves.helper.ts

```diff
@@ -0,0 +1,128 @@
+import { HeatMapDTO, Leave, LeaveHeatMapDTO, LeavesDTO, LeavesTableDTO } from '@/graphql/graphql';
+
+export function mapToTableDTOs(leaves: Leave[]): LeavesTableDTO[] {
+  return leaves.map((leave) => ({
+    date: leave.leaveDate,
+    createdAt: leave.createdAt,
+    leaveTypeId: leave.leaveTypeId,
+    isWithPay: leave.isWithPay,
+    reason: leave.reason ?? '',
+    status: leave.isDeleted ? 'CANCELLED' : 'APPROVED', // or however you define status
```

Comment:
```suggestion
    status: leave.isDeleted ? 'CANCELLED' : 'APPROVED', // or however you define status
```

Please delete unnecessary code comments